lib/runtime/wazero/instance_test.go

```diff
@@ -1096,7 +1095,7 @@ func newTrieFromPairs(t *testing.T, filename string) trie.Trie {
 		entries[pairArr[0].(string)] = pairArr[1].(string)
 	}
 
-	tr, err := inmemory.LoadFromMap(entries, trie.V0)
```

Comment:
unnecessary diff

Reply:
I see this comment on the previous PR (lost when we combined them) https://github.com/ChainSafe/gossamer/pull/3865#discussion_r1552423920

So maybe there is context to this change I am missing. @dimartiro could you add some context here? else I will delete this diff

Reply:
This is just to remove the old import and use the new one with the alias

Reply:
okay, i will keep then